src/markdown-pages/build-apps/build-hello-world-app.mdx

```diff
@@ -20,13 +20,14 @@ We also have a 5-minute video that covers the steps below.
 
 To get started, make sure you have accounts in GitHub and [New Relic](https://newrelic.com/signup/).
 
-Use your New Relic account credentials to log into our [CLI wizard](https://one.newrelic.com/launcher/developer-center.launcher?pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==). It sets you up with the following:
+To develop projects, you need our New Relic One CLI (command line interface). 
 
-* New Relic API key
-* Download options for the New Relic CLI (command line interface)
-* Customized CLI commands to configure your environment
+If you haven't already installed it, do the following:
 
-After you install the CLI and configure your environment, continue with the steps below. If you want to build your app and see it locally, you need to install [Node.js](https://nodejs.org/en/download/) and run `npm install`.
+* Install [Node.js](https://nodejs.org/en/download/).
+* Complete all the steps in our [CLI wizard](https://one.newrelic.com/launcher/developer-center.launcher?pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==).
+
+For additional details about setting up your environment, see [Set up your development environment](https://developer.newrelic.com/build-apps/set-up-dev-env).
```

Comment:
should be: see [Set up your development environment](/build-apps/set-up-dev-env/).

same here - link is (/build-apps/set-up-dev-env)